docs/sources/setup/install/helm/deployment-guides/gcp.md

```diff
@@ -0,0 +1,625 @@
+  --location=<REGION> \
+  --default-storage-class=STANDARD \
+  --public-access-prevention \
+  --uniform-bucket-level-access \
+  --soft-delete-duration=7d
+```
+
+Make sure to replace the `region` and `bucket` name with your desired values. We will revisit the bucket policy later in this guide.
+
+Here's an example with all the variables filled in:
+
+```bash
+gcloud storage buckets create gs://loki-gcp-chunks gs://loki-gcp-ruler \
+  --location=europe-west4 \
+  --default-storage-class=STANDARD \
+  --public-access-prevention \
+  --uniform-bucket-level-access \
+  --soft-delete-duration=7d
+```
+
+When you run this command, you should get something like this in response:
+
+```bash
+Creating gs://loki-gcp-chunks/...
+Creating gs://loki-gcp-ruler/...
+```
+
+
+## Defining IAM roles and policies
+
+IAM determines who can access which resources on GCP and can be configured in several ways. The recommended method for allowing Loki to access GCS is to use Workload Identity Federation. This method is more secure than using creating and distributing a service account key. The below steps show how to create the role and policy using the gcloud CLI.
+
+### Authenticating to the GKE cluster
+
+You need to be able to run `kubectl` commands on the cluster, so make sure you have it installed (run `gcloud components install kubectl` if not) and then run this command:
+
+```bash
+gcloud container clusters get-credentials <CLUSTER_NAME> \
+  --region=<REGION>
+```
+
+Here's an example of that command with the variables filled in:
+
+```bash
+gcloud container clusters get-credentials loki-gcp \
+  --region=europe-west4
+```
+
+This will authenticate you via your GCP IAM identity, write the cluster's access info to your local kubeconfig (usually `~/.kube/config`), and then allow `kubectl` commands talk to the right cluster from now on.
+
+Then check that you're connected to the GKE cluster and that you're accessing it via `kubectl` by running:
+
+```bash
+kubectl config current-context
+```
+
+You should get something like this in return:
+
+```bash
+gke_my-project-123456_europe-west4_loki-gcp
+```
+
+### Create a Kubernetes Namespace
+
+Create a K8s namespace where you'll install your Loki workloads:
+
+```bash
+kubectl create namespace <NAMESPACE>
+```
+
+Replace `<NAMESPACE>` with the namespace where your Loki workloads will be located.
+
+Example:
+
+```bash
+kubectl create namespace loki
+```
+
+You should get the output:
+
+```bash
+namespace/loki created
+```
+### Create Kubernetes Service Account (KSA)
+
+A KSA is a cluster identity (service account, named `default` by default) assigned to pods that allows pods to interact with each other.
+
+Create a KSA on your K8s cluster:
+
+```bash
+kubectl create serviceaccount <KSA_NAME> \
+  --namespace <NAMESPACE>
+```
+
+Replace `<KSA_NAME>` with the name of the KSA created above, and `<NAMESPACE>` with the namespace where your Loki/GEL workloads are located.
```

Comment:
I think this is the only place (or one of two) where you say "Loki/GEL". Do you want to just say Loki, or go back through and say Loki/GEL everywhere?

Reply:
Fixed! Removed "/GEL" and just said Loki in two instances.